applications/nrf5340_audio/src/audio/streamctrl_unicast_server.c

```diff
@@ -0,0 +1,722 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "streamctrl.h"
+
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/device.h>
+#include <zephyr/debug/stack.h>
+#include <zephyr/zbus/zbus.h>
+
+#include "nrf5340_audio_common.h"
+#include "led.h"
+#include "button_assignments.h"
+#include "macros_common.h"
+#include "audio_system.h"
+#include "button_handler.h"
+#include "data_fifo.h"
+#include "le_audio.h"
+#include "bt_mgmt.h"
+#include "bt_rend.h"
+#include "audio_datapath.h"
+#include "bt_content_ctrl.h"
+#include "unicast_server.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(streamctrl_unicast_server, CONFIG_STREAMCTRL_LOG_LEVEL);
```

Comment:
```suggestion
LOG_MODULE_REGISTER(streamctrl, CONFIG_STREAMCTRL_LOG_LEVEL);
```
For consistency

Reply:
Discuss

Reply:
I will change the other ones to reflect this specific version.